src/clj/game/cards/agendas.clj

```diff
@@ -341,30 +341,33 @@
                :effect (effect (damage eid :meat (meat-damage state card) {:card card}))}})
 
    "Clone Retirement"
-   {:msg "remove 1 bad publicity" :effect (effect (lose :bad-publicity 1))
+   {:msg "remove 1 bad publicity"
+    :effect (effect (lose :bad-publicity 1))
     :silent (req true)
     :stolen {:msg "force the Corp to take 1 bad publicity"
              :effect (effect (gain-bad-publicity :corp 1))}}
 
    "Corporate Sales Team"
-   (let [e {:effect (req (when (pos? (get-counters card :credit))
-                           (gain-credits state :corp 1)
-                           (system-msg state :corp (str "uses Corporate Sales Team to gain 1 [Credits]"))
-                           (add-counter state side card :credit -1)))}]
+   (let [ability {:silent (req true)
+                  :counter-cost [:credit 1]
+                  :msg "gain 1 [Credit]"
+                  :effect (effect (gain-credits :corp 1))}]
      {:effect (effect (add-counter card :credit 10))
```

Comment:
This card should probably have an ability like PAD Campaign so it can be ordered before cards like Executive Boot Camp.

Reply:
Good call, I'll do that.